models/workload.go

```diff
@@ -39,6 +39,15 @@ type WorkloadList struct {
 	Validations IstioValidations `json:"validations"`
 }
 
+type LogType string
+
+const (
+	AppLog      LogType = "app"
+	ProxyLog    LogType = "proxy"
+	WaypointLog LogType = "waypoint"
+	ZtunnelLog  LogType = "ztunnel"
```

Comment:
This is a totally subjective comment, so please
 feel free to ignore.  I like to name these more like an enum, so LogTypeApp, LogTypeProxy, etc...